Modules/Core/Controllers/ReportsController.php

```
@@ -37,79 +53,102 @@ public function salesByClient(Request $request): \Illuminate\Contracts\View\View
+     * @legacy-file application/modules/reports/controllers/Reports.php
      */
     public function paymentHistory(Request $request): \Illuminate\Contracts\View\View
     {
         if ($request->input('btn_submit')) {
-            $results = (new ReportsService())->paymentHistory($request->input('from_date'), $request->input('to_date'));
-            $data    = [
+            $results = $this->reportsService->paymentHistory(
+                $request->input('from_date'),
+                $request->input('to_date')
+            );
+            $data = [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (undefined)

(UnusedLocalVariable)